src/main/java/org/jenkinsci/plugins/github_branch_source/ApiRateLimitChecker.java

```diff
@@ -103,7 +103,7 @@ public void checkApiRateLimit(@NonNull TaskListener listener, GitHub github) thr
 
                 )));
                 listener.getLogger().println(GitHubConsoleNote.create(System.currentTimeMillis(),
-                        "Jenkins is using \"Throttle on Over\" to restrict GitHub API requests only when near or above the rate limit. Use \"Throttle for Normalize\" to try and evenly distribute GitHub API requests."));
+                        "Jenkins is restricting GitHub API requests only when near or above the rate limit. To configure a different rate limiting strategy, such as having Jenkins restrict GitHub API requests only when near or above the GitHub rate limit, go to \"GitHub API usage\" under \"Configure System\" in the Jenkins settings."));
```

Comment:
```suggestion
                        "Jenkins is restricting GitHub API requests only when near or above the rate limit. To configure a different rate limiting strategy, such as having Jenkins attempt to evenly distribute GitHub API requests, go to \"GitHub API usage\" under \"Configure System\" in the Jenkins settings."));
```

Reply:
thanks, updated in e74566beb93b0a098665cd5f44d764707a5b36eb